src/sched.rs

```diff
@@ -197,15 +198,16 @@ pub fn sched_setaffinity(pid: isize, cpuset: &CpuSet) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
-pub fn clone(mut cb: CloneCb, stack: &mut [u8], flags: CloneFlags) -> Result<pid_t> {
+pub fn clone(mut cb: CloneCb, stack: &mut [u8], flags: CloneFlags, signal: Option<c_int>) -> Result<pid_t> {
```

Comment:
I would prefer to let signal remain of type `c_int`, the type for signals. In order to address your (valid) concern, we could add a check into our wrapper, that signals an appropriate error.